### ISSUE TITLE
Save the best model

### ISSUE THREAD

**Issue by FranciscoPereira9** [FranciscoPereira9/FaceMaskDetection-PyTorch#3]
***Context***: The model saved is the one resulting from the last epoch. Sometimes the best model doesn't occur at the end.

***Idea***: Save the model with minimum loss on all training epochs.

***Solution***: Keep track of loss values through different epochs and save the best one in a file.
